lib/tasks/import_data_to_bigquery.rake

```diff
@@ -0,0 +1,24 @@
+DEFAULT_BATCH_SIZE = 1
```

Comment:
Is this right? Baby steps? 😛

Reply:
We ran with a batch size of 200 and no sleeps and we didn't even reach 50% of the redis memory running with 12 worker instances. The risk I was talking about yesterday with running it so slowly is it's annoying to restart if the container instance recycled or it gets interrupted for whatever reason. We've run a partial import today and it wasn't 'that' painful but still.

Reply:
Defaulted to 200